V2 App24

App24 opened the pull request:
Files I've created/modified

App24 committed:
Added extra events

Added Sector Creation and Destruction Events, Virtual World Destruction Events, and Lyoko Warrier Dexanification Event.

GoodOldJack12 commented on LyokoAPI/Events/SectorCreationEvent.cs:
```diff
@@ -0,0 +1,65 @@
+    public class SectorCreationEvent
+    {
+        private static event Events.OnSectorEvent SectorE;
+        public static void Call(ISector sector)
```
There should also be a way of calling this with a string. See the tower activation event for examples.

GoodOldJack12 commented on LyokoAPI/Events/SectorDestructionEvent.cs:
```diff
@@ -0,0 +1,65 @@
+    public class SectorDestructionEvent
+    {
+        private static event Events.OnSectorEvent SectorE;
+        public static void Call(ISector sector)
```
There should also be a way of calling this with a string. See the virtualization events for examples.

GoodOldJack12 commented on LyokoAPI/Events/VirtualWorldDestructionEvent.cs:
```diff
@@ -0,0 +1,65 @@
+    class VirtualWorldDestructionEvent
+    {
+        private static event Events.OnVirtualWorldEvent VirtualWorldE;
+        public static void Call(IVirtualWorld world)
```
There should also be a way of calling this with a string. See the virtualization events for examples.

GoodOldJack12 commented on LyokoAPI/VirtualEntities/LyokoWarrior/LyokoWarrior.cs:
```diff
@@ -95,6 +95,14 @@ internal LyokoWarrior Xanafy()
             return this;
         }
 
+        internal LyokoWarrior Dexanafy()
```
This is not a comment on your code but we might not have to do the location changes in these type of methods because IFSCL may call LW_MoveEvent. Since the rest of the file also manually changes the location though, you dont have to change this now.

GoodOldJack12 reviewed (changes requested):
Seems fine in general, just a few tweaks and we can merge

App24 committed:
Modified Events

Modified events for sector creation and destruction and events for virtual world destruction.

App24 committed:
Changed .Net Framework

Changed .Net Framework to fit with the original project

GoodOldJack12 committed:
A sector is more likely to have 10 towers than none

GoodOldJack12 committed:
Removed redundant call parameters

The sectors dont add value to the event

GoodOldJack12 committed:
Sector is more likely to contain 10 towers than none

GoodOldJack12 committed:
Removed redundant call parameters

The towers in a sector don't add value to the event

GoodOldJack12 reviewed (approved):
Seems fine now
